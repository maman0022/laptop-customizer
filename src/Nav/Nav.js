import React from 'react';
import './Nav.css'

export default function Nav() {
  return (
    <header>
      <h1>ELF Computing | Laptops</h1>
    </header>
  )
}