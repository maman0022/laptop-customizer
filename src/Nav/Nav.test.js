import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Nav from './Nav'

describe('Nav Component',()=>{
  it('the Nav with a h1 element',()=>{
    const wrapper=shallow(<Nav/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})