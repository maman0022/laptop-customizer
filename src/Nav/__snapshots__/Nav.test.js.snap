// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component the Nav with a h1 element 1`] = `
<header>
  <h1>
    ELF Computing | Laptops
  </h1>
</header>
`;
